addons/danger/functions/fnc_brain.sqf

```diff
@@ -111,9 +111,10 @@ private _group = group _unit;
 // Enemy Near
 if (
     _dangerCause isEqualTo DANGER_ENEMYNEAR
-    || { !isNull _dangerCausedBy }
-    && { (_group getVariable [QGVAR(contact), 0]) < time }
-    && { !(_group getVariable [QGVAR(disableGroupAI), false]) }
+    || {
+        !isNull _dangerCausedBy 
+        && { (_group getVariable [QGVAR(contact), 0]) < time }
```

Comment:
Why was the check for `QGVAR(disableGroupAI)` removed?

Reply:
Because it had no function. At one point this section called information sharing-- which could trigger combat mode in nearby groups.